arches/app/templates/views/components/widgets/datepicker.htm

```diff
@@ -72,7 +74,7 @@
         type="" 
         id="" 
         data-bind="
-            attr: {placeholder: $root.translations.viewMode},
+            attr: {placeholder: $root.translations.viewMode, 'aria-label': $root.translations.viewMode},
```

Comment:
there is no translation for this label.  Add this to the translations section in javascript.htm
```view-mode='{% trans "Date Picker View Mode" as viewMode %} "{{ viewMode|escapejs }}"'```